src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -63,8 +64,8 @@ def get_monorail_issuetracker_issue_id_dictionary(file_loc):
     fieldnames = ['monorail_id', 'issuetracker_id']
     reader = csv.DictReader(csvfile, fieldnames=fieldnames)
     for row in reader:
-      monorail_issue_id = row[fieldnames[0]]
-      issuetracker_issue_id = row[fieldnames[1]]
-      issue_id_dictionary[monorail_issue_id] = issuetracker_issue_id
+      key_id = row[fieldnames[1]] if roll_back else row[fieldnames[0]]
+      value_id = row[fieldnames[0]] if roll_back else row[fieldnames[1]]
+      issue_id_dictionary[key_id] = value_id
 
-  return issue_id_dictionary  # { monorail_issue_id: issuetracker_issue_id }
+  return issue_id_dictionary  # { key_id: value_id }
```

Comment:
This comment is now superfluous: keys map to values :)

Reply:
Done